### PR TITLE
Assignment 2: Convert to a plugin

### PR DESCRIPTION
**Pull request by ipoddubny**

### CODE REVIEW

File: lib/version.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Load modules
+
+var Package = require('../package.json');
+
+
+// Declare internals
+
+var internals = {};
```
Comment (edwin-bluekite) [inline lib/version.js]: internals is declared but not used

Comment (TheAlphaNerd) [inline lib/version.js]: prior art --> https://github.com/hueniverse/hueniversity/blob/master/lib/index.js#L10

Comment (hueniverse) [inline lib/version.js]: `internals` should always be declared. Saves you from forgetting later.

File: lib/version.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                return reply({ version: Package.version });
+            }
+        }
+    });
+
+    return next();
+};
+
+exports.register.attributes = {
+    name: '/version route handler'
```
Comment (hueniverse) [inline lib/version.js]: `name` should be access friendly (e.g. no spaces, or characters you need to put in quotes).

**Pull request closed without merging**